lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -1487,17 +1487,7 @@ postTrezorWallet
     => ctx
     -> ByronWalletPostData '[12,15,18,21,24]
     -> Handler ApiByronWallet
-postTrezorWallet ctx body = do
-    postLegacyWallet ctx (rootXPrv, pwd) $ \wrk wid ->
-        W.createIcarusWallet @s genesisParams wrk wid wName
-            (RootCredentials rootXPrv pwdP)
-  where
-    wName    = body ^. #name . #getApiT
-    seed     = body ^. #mnemonicSentence . #getApiMnemonicT
-    pwd      = body ^. #passphrase . #getApiT
-    pwdP     = preparePassphrase currentPassphraseScheme pwd
-    rootXPrv = Icarus.generateKeyFromSeed seed pwdP
-    genesisParams = ctx ^. #netParams
+postTrezorWallet = postIcarusWallet
```

Comment:
I'm surprised, but it looks correct 😮

Reply:
I wonder what the difference is then. I see some talk of "Icarus vs Icarus-Trezor" e.g. here

https://www.reddit.com/r/TREZOR/comments/tvssuy/question_about_ada_derivation_path_and_changes/

Reply:
Maybe it could have been just a difference in allowed mnemonic lengths 🤷‍♂️ https://github.com/cardano-foundation/cardano-wallet/pull/1236/files#diff-43aba9d41169fbb6946ae3d716c73bcd3b431a7b42bde0163a3b00b976bbb6e3R325